src/com/hms_networks/americas/sc/taginfo/TagInfoEnumeratedIntToString.java

```diff
@@ -0,0 +1,80 @@
+package com.hms_networks.americas.sc.taginfo;
+
+import java.util.ArrayList;
+
+public class TagInfoEnumeratedIntToString extends TagInfo {

```

Comment:
class missing javadoc. 

You can probably just copy/paste the javadocs from the regular TagInfo class and just tweak it for this.